src/common/pico_sync/include/pico/critical_section.h

```diff
@@ -7,6 +7,7 @@
 #ifndef _PICO_CRITICAL_SECTION_H
 #define _PICO_CRITICAL_SECTION_H
 
+#include "pico/platform/compiler.h"
```

Comment:
not a fan of this - it should be included by `pico.h`

Reply:
np. Didn't know what the right include was.